Libs/MRML/Core/vtkMRMLScene.cxx

```diff
@@ -1250,6 +1250,23 @@ vtkMRMLNode* vtkMRMLScene::AddNode(vtkMRMLNode *n)
 }
 
 //------------------------------------------------------------------------------
+vtkMRMLNode* vtkMRMLScene::AddNewNodeByClass(std::string className, std::string nodeName)
```

Comment:
It helps if you indicate default argument values in the cxx file - for example
 ...AddNewNodeByClass(std::string className, std::string nodeName /* ="" */)